modules/services/model/schemas/au.com.shiftyjelly.pocketcasts.models.db.AppDatabase/118.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 118,
-    "identityHash": "49d05d7d0fb9ebc814308b894bad9900",
+    "identityHash": "c4c18ea275279a868fab5496462fa977",
```

Comment:
This is a safe change because there wasn't release of `7.97` yet so migration is not needed.